app/code/UAE/GroupedProduct/view/frontend/layout/catalog_product_view_type_grouped.xml

```diff
@@ -32,5 +32,8 @@
         <referenceContainer name="product.info.details" remove="true" />
         <referenceContainer name="product.info.review" remove="true" />
         <move element="product.info" destination="content" before="product.info.details" />
+        <referenceContainer name="before.body.end">
+            <block class="Magento\Framework\View\Element\Template" name="grouped.product.bottom.bar" template="UAE_GroupedProduct::product/bottom-add-to-cart.phtml" />
+        </referenceContainer>
```

Comment:
@Andrewb89 - I wanted to give some FE feedback here: Since the add-to-cart element is getting fixed to the bottom of the window, we didn't actually need to add a block in layout or create a new template. I was able to get the result simply by using position:fixed in the CSS for div.box-tocart. ([See here on stage.](https://staging.unitednow.com/catalog/product/view/id/1467))

Reply:
(Note that I already did this in a later commit, so you don't need to change anything this time. I just wanted you to know for next time.)